src/rbus/rbus_handle.h

```diff
@@ -102,6 +103,7 @@ struct _rbusHandle
   rbusValueChangeDetector_t valueChangeDetector;
 
   uint32_t              busyWaitSleepDurationMillis;
+  atomic_int            reentrancyGuard;
```

Comment:
move the whole structure definition into .c file and the inclusion of header file also limit to .c file..

as the whole library is .c based, the gcc will take care of using proper definition of atomic